### PR TITLE
Add SRA user manual

### PR DESCRIPTION
**Pull request by apmasell**
JIRA: GLT-2538

### CODE REVIEW

File: docs/user_manual/sra.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+An experiment connects a study to a library on a sequencing platform. Select
+the appropriate library from the dialog box. A second dialog box will appear
+with all the studies that belong to the project of this library. Select the
+appropriate library and provide the title and alias of the experiment. The
+alias and title map directly to the alias and title in the [ENA
+model](https://ena-docs.readthedocs.io/en/latest/prog_04.html#create-the-run-and-experiment-xml).
+
+Once an experiment exists, all sequencer runs of this library from the same
+platform are connected with this experiment.
```
Comment (callunity) [inline docs/user_manual/sra.md]: This doesn't seem to be entirely accurate --  I made [this experiment|https://miso.gsi.oicr.on.ca/miso/experiment/2] for the library listed in the experiment title, but that library is also on another run which doesn't show up in the experiment. It does seem to capture all the sequencing lanes of this library from the same run + platform though... perhaps this is a MISO bug, if an SRA/ENA experiment is supposed to capture all runs for a given library?

File: docs/user_manual/sra.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+Experiments can be seen in several places:
+
+* From the "SRA" section on the left, choose "Experiments" to view all
+  experiments. This shows all experiments.
+* From the "Tracking" section on the left, choose "Studies", then select the study
+  of interest. This shows experiments associated with this study.
+* From the "Tracking" section on the left, choose "Sequencer Runs", then select the
+  run of interest. This shows experiments associated with this sequencing run.
+* From the "Tracking" section on the left, choose "Libraries", then select the
+  library of interest. This shows experiment associated with this library.
```
[the remark this answers is not in the document]
Comment (apmasell) [reply docs/user_manual/sra.md]: I'm confused. Those tables should filter to show relevant experiments.

File: docs/user_manual/sra.md (hunk 1 above)
Comment (apmasell) [reply docs/user_manual/sra.md]: Docs are wrong. I will fix.

File: docs/user_manual/sra.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (callunity) [reply docs/user_manual/sra.md]: "This shows" has me expecting that this is the main thing that is seen when this link is clicked; in all cases here, the user is taken to another page that . The page has an Experiments table on it, but they have to go looking/scrolling to find it.

File: docs/user_manual/ena.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
+formerly ERA; previously connected to the SRA. The archive requires not only
+the sequencing data, but the biological metadata associated with the data. To
 facilitate this processes, MISO provides a mechanism to export the appropriate
 metadata required. This requires adding some extra information to MISO during
 the sequencing process.
 
 
 {% assign sub = 1 %}
-{% include userman-heading.md section=section sub=sub section-title=section-title title="Studies" %}
+{% include userman-heading.md section=section sub=sub page.section=page.section title="Studies" %}
```
Comment (callunity) [inline docs/user_manual/ena.md]: Invalid syntax: `page.section=page.section` -> `section=page.section`

File: docs/user_manual/ena.md [hunk 4]
```diff
@@ -1,29 +1,29 @@
 ---
 layout: page
-title: "24. SRA/ERA/ENA Support"
+title: "24. European Nucleotide Archive Support"
```
[the remark this answers is not in the document]
Comment (apmasell) [reply docs/user_manual/ena.md]: I didn't know that's how that worked. Fixed.

File: core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java [hunk 5]
```diff
@@ -45,7 +45,7 @@
 import uk.ac.bbsrc.tgac.miso.core.data.type.HealthType;
 import uk.ac.bbsrc.tgac.miso.core.data.type.SubmissionActionType;
 
-public class EraSubmissionPreparation {
+public class EnaSubmissionPreparation {
   private abstract class ChildSubmissionFile {
```
Comment (sqwci) [inline core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java]: MINOR Convert the abstract class "ChildSubmissionFile" into an interface. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)

**Comment by sqwci** [comment]
SonarQube analysis reported 17 issues
* ![CRITICAL][CRITICAL] 10 critical
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [EnaSubmissionPreparation.java#L81](https://github.com/TGAC/miso-lims/blob/1d19abb9ed1b9a5fdc5fac8569349c6ebf7c40dc/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java#L81): Define a constant instead of duplicating this literal "alias" 5 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [EnaSubmissionPreparation.java#L87](https://github.com/TGAC/miso-lims/blob/1d19abb9ed1b9a5fdc5fac8569349c6ebf7c40dc/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java#L87): Define a constant instead of duplicating this literal "center_name" 4 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [EnaSubmissionPreparation.java#L91](https://github.com/TGAC/miso-lims/blob/1d19abb9ed1b9a5fdc5fac8569349c6ebf7c40dc/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java#L91): Define a constant instead of duplicating this literal "refname" 4 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [EnaSubmissionPreparation.java#L92](https://github.com/TGAC/miso-lims/blob/1d19abb9ed1b9a5fdc5fac8569349c6ebf7c40dc/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java#L92): Define a constant instead of duplicating this literal "refcenter" 5 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [EnaSubmissionPreparation.java#L129](https://github.com/TGAC/miso-lims/blob/1d19abb9ed1b9a5fdc5fac8569349c6ebf7c40dc/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java#L129): Refactor this method to reduce its Cognitive Complexity from 22 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)
1. ![CRITICAL][CRITICAL] [EnaSubmissionPreparation.java#L133](https://github.com/TGAC/miso-lims/blob/1d19abb9ed1b9a5fdc5fac8569349c6ebf7c40dc/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java#L133): Define a constant instead of duplicating this literal "TITLE" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [EnaSubmissionPreparation.java#L141](https://github.com/TGAC/miso-lims/blob/1d19abb9ed1b9a5fdc5fac8569349c6ebf7c40dc/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java#L141): Define a constant instead of duplicating this literal "accession" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [EnaSubmissionPreparation.java#L382](https://github.com/TGAC/miso-lims/blob/1d19abb9ed1b9a5fdc5fac8569349c6ebf7c40dc/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java#L382): Secure this "Transformer" by either disabling external DTDs or enabling secure processing. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4435)
1. ![CRITICAL][CRITICAL] [EnaSubmissionPreparation.java#L472](https://github.com/TGAC/miso-lims/blob/1d19abb9ed1b9a5fdc5fac8569349c6ebf7c40dc/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java#L472): Define a constant instead of duplicating this literal "SUBMISSION_SET" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [EnaSubmissionPreparation.java#L482](https://github.com/TGAC/miso-lims/blob/1d19abb9ed1b9a5fdc5fac8569349c6ebf7c40dc/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java#L482): Secure this "Transformer" by either disabling external DTDs or enabling secure processing. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4435)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: docs/user_manual/european_nucleotide_archive_support.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+{% include userman-heading.md sub=sub subsub=subsub section=page.section title="Studies List" %}
+
+The studies list can be accessed by clicking the "Studies" link from the Tracking list at the left side of the
+screen. The page shows all of the studies from all projects. At the top of the
+table, you will find controls for adding, and deleting studies.
+
+You can click on a study name or alias to get to the Edit Study page.
+
+For a particular project, the studies can be found on the [project
+page](projects.html#edit_project_page). Click the "My Projects" tab near the
```
Comment (djcooke) [inline docs/user_manual/european_nucleotide_archive_support.md]: * "[Edit Project page]"
* This section should not contain instructions to get there (My Projects tab, etc)

File: docs/user_manual/european_nucleotide_archive_support.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+  run of interest. This shows experiments associated with this sequencing run.
+* From the "Tracking" section on the left, choose "Libraries", then select the
+  library of interest. This shows experiment associated with this library.
+
+{% assign subsub = subsub | plus: 1 %}
+{% include userman-heading.md sub=sub subsub=subsub section=page.section title="Creating an Experiment" %}
+
+From the "Tracking" section on the left, choose "Sequencer Runs", and then
+select the sequencer run of interest. If pools have not already been assigned
+to lanes, assign them. From the "Experiment" table, choose "Create New".
```
[the remark this answers is not in the document]
Comment (djcooke) [reply docs/user_manual/european_nucleotide_archive_support.md]: bump

**Comment by djcooke** [comment]
Tried to bump one of my comments above from first review, but GitHub is still hiding it for some reason (in the ~~~ 1 hidden conversation ~~~ block)

File: docs/user_manual/european_nucleotide_archive_support.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (apmasell) [reply docs/user_manual/european_nucleotide_archive_support.md]: Fixed the second. The first...the orders section is full of lanes. We don't consistently use partitions. I don't know. @callunity?

[the remark this answers is not in the document]
Comment (callunity) [reply docs/user_manual/european_nucleotide_archive_support.md]: Mild preference for changing all lanes -> partitions...

**Pull request merged**